bootstrap/src/lib/util.ts

```diff
@@ -24,6 +24,14 @@ export const toObjectWithNumbers = (obj: any) => {
   return Object.fromEntries(Object.entries(obj).map(([k, v]) => [k, toNumber(v)]))
 }
 
+// pick a random string from env var after splitting with the delimiter ("a&b&c" "&" -> choice(["a","b","c"]))
+export const getRandomEnv = (name: string, delimiter = ',', prefix = '') => {
```

Comment:
I left a comment in the batch PR about this -> https://github.com/smartcontractkit/external-adapters-js/pull/226#discussion_r570260831

What do you think?